Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-findOneAndDelete.txt

MongoDB\\Collection::findOneAndDelete()
=======================================

.. meta::

>    :description: Use `findOneAndDelete()` to find and delete a single document matching a query in a MongoDB collection, with options for collation, projection, and more.

Reviewer: ```suggestion
   :description: Find and delete a single document in a collection with the MongoDB PHP Library, with options for collation, projection, and more.
```
